### ISSUE TITLE
Choose a logging crate

### ISSUE THREAD

**Issue by Nyefari** [CylonSB/bounded-planet#24]
Tracing or Log

**Comment by martindevans** [comment]
Per discussion in voip the other day `tracing` seems to do everything we want and works properly with async stuff.

**Comment by Nyefari** [comment]
I agree. I vote `tracing` and to tie in `log` if we find we need it.
